Packs/Cyberpion/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Cyberpion",
+    "description": "Attack surface discovery and vulnerability management",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Cyberpion",
+    "url": "",
+    "email": "isaac@cyberpion.com",
```

Comment:
Better to fill in here URL of your support, and not your personal one.
Also, recommend to fill in the URL